src/server/services/tips/tips-service.ts

```diff
@@ -34,6 +34,17 @@ export function convertTipToNotication(tip: MyTip): MyNotification {
   } as MyNotification;
 }
 
+export function prefixTipNotification(
+  notification: MyNotification
+): MyNotification {
+  return {
+    ...notification,
+    title: `${!notification.title.startsWith('Tip: ') ? 'Tip: ' : ''}${
```

Comment:
Is 't niet beter om iets uitgebreider te checken of Tip: ervoor staat? Mogelijk zet men er met de tijd dingen neer als:

```
tip: dingen
Tip:Dingen
tip:dingen
```

Je zou een regex kunnen gebruiken. 

```
const pattern = /^tip(:?\s?)/i;
for (const str of [
  'tip:test',
  'tip: test',
  'Tip:test',
  'tip test',
  'Tip test',
]) {
  const result = pattern.test(str);
  console.log(str, result, str.replace(pattern, 'Tip: '));
}
```